ouroboros-consensus/src/Ouroboros/Consensus/Demo/Run.hs

```diff
@@ -1,117 +1,51 @@
 {-# LANGUAGE FlexibleContexts        #-}
-{-# LANGUAGE MultiParamTypeClasses   #-}
-{-# LANGUAGE RankNTypes              #-}
-{-# LANGUAGE TypeFamilies            #-}
+{-# LANGUAGE FlexibleInstances       #-}
 {-# LANGUAGE UndecidableInstances    #-}
 {-# LANGUAGE UndecidableSuperClasses #-}
-
 -- | Infrastructure required to run the demo
---
--- The definitions in this module are independent from any specific protocol.
-module Ouroboros.Consensus.Demo.Run (
-    -- * ProtocolInfo
-    ProtocolInfo(..)
-  , NumCoreNodes(..)
-    -- * Constraints required to run the demo
-  , DemoHeaderHash(..)
-  , RunDemo(..)
+module Ouroboros.Consensus.Demo.Run
+  ( RunDemo(..)
```

Comment:
This could be renamed to `RunProtocol`, but it's probably good to do that in a separate step / PR.